examples/simple/webpack.config.js

```diff
@@ -66,6 +66,22 @@ module.exports = {
                 'ra-input-rich-text',
                 'src'
             ),
+            'ra-tree-core': path.join(
```

Comment:
aren't you missing the translation packages?

Reply:
Added